test/integration/lib/test-actions/index.ts

```diff
@@ -119,8 +119,11 @@ async function _startAndAcceptInteraction(
   assert(interactId)
 
   // Accept
+  const url = new URL(senderWalletAddress.authServer)
+  url.port = '3109'
+
   const acceptResponse = await fetch(
-    `${senderWalletAddress.authServer}/grant/${interactId}/${nonce}/accept`,
+    `${url.toString()}grant/${interactId}/${nonce}/accept`,
```

Comment:
I'm not sure the interaction url should be derived from the auth server url 🤔

Reply:
I mean, this is used only for integration tests, and tests are designed so that on the same server, there is auth server running and interaction server running, so it should not be a problem for tests. But for ASE, they will need to specify this address in their config I assume

Reply:
Fair enough 🙂